python-bindings/ARToolKitNFT_py.h

```diff
@@ -2,6 +2,7 @@
 #include <AR/ar.h>
 #include <pybind11/pybind11.h>
 #include <pybind11/numpy.h>
+#include <pybind11/stl.h>
```

Comment:
this is needed for the addNFTMarkers method, otherwise you will get an error while loading an arry of path strings.